addons/website_event/views/event_event_views.xml

```diff
@@ -9,7 +9,19 @@
         <field name="arch" type="xml">
             <xpath expr="//group[@name='right_event_details']/field[@name='company_id']" position="after">
                 <field name="website_id" invisible="1"/>
+                <field name="website_published" invisible="1"/>
                 <field name="website_id" options="{'no_create': True}" domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]" groups="website.group_multi_website"/>
+                <label for="visibility" string="Visibility"/>
```

Comment:
Should we call it "Website Visibility" ?

Maybe it's made clear in the context of the field based on the surrounding fields, not sure.